### ISSUE TITLE
immutableTypes does not make arguments readonly 

### ISSUE THREAD

**Issue by lirbank** [dotansimha/graphql-code-generator#1488]
**Describe the bug**
Input types are not set to `readonly`. All other types seem to be correctly marked `readonly`/`ReadonlyArray`.

Input:
```graphql
type Query {
  user(id: ID!): User
}
```

Output:
```ts
export interface UserQueryArgs {
  id: string;
}
```

Expected:
```ts
export interface UserQueryArgs {
  readonly id: string;
}
```

**Comment by dotansimha** [comment]
Hi @lirbank ! Thank you for reporting it. 
We are currently working on a big refactor for the TypeScript plugins, and preparing for a 1.0 release. 
I fixed it here: https://github.com/dotansimha/graphql-code-generator/pull/1484/commits/c8b59c8737fdc9e68dd8dbb3b8640ef4d64b7aaf 
It will be available soon.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
